static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js

```diff
@@ -72,35 +76,54 @@ define([
 
     function showAdblockBanner() {
         var contributors = history.getContributors(),
-            allVariations = [{
-                supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_MONBIOT',
-                quoteText: 'Become a Guardian Member and support independent journalism',
-                quoteAuthor: 'George Monbiot',
-                customCssClass: 'monbiot',
-                imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/contributor/2015/7/9/1436429159376/George-Monbiot-L.png?w=300&amp;q=85&amp;auto=format&amp;sharp=10&amp;s=903233b032379d7529d7337b8c26bcc9'
-            },
-            {
-                supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_MUIR',
-                quoteText: 'Support and become part of the Guardian',
-                quoteAuthor: 'Hugh Muir',
-                customCssClass: 'muir',
-                imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/pictures/2014/3/13/1394733739000/HughMuir.png?w=300&amp;q=85&amp;auto=format&amp;sharp=10&amp;s=c1eeb35230ad2a215ec9de76b3eb69fb'
+            contributorBannersUK = [
```

Comment:
I would keep the `contributorBanners` edition-independent and turn them into UK banners with the internationalize function you're using for the others.